src/pytket_pecos/emulator.py

```diff
@@ -6,7 +6,7 @@
 from pytket.circuit import Circuit
 from pytket.phir.api import pytket_to_phir
 from pytket.utils.outcomearray import OutcomeArray
-from pytket.wasm import WasmFileHandler
+from pytket.wasm.wasm import WasmModuleHandler
```

Comment:
This might warrant a vanity import from pytket, but would require changes there and a version bump.